website/content/docs/oss/installing/dev-mode.mdx

```diff
@@ -0,0 +1,64 @@
+---
+layout: docs
+page_title: What is Dev Mode?
+description: Getting started with Boundary OSS in Dev mode
+---
+
+# What is Dev Mode?
+
+Before getting started with Boundary OSS, it's important to understand a few key
+concepts. Please consult our [Boundary concepts](/docs/concepts) page to
+familiarize yourself with Boundary's architecture, and terminology.
+
+Dev mode is an all-in-one installation method for getting started with Boundary
+quickly. As the name implies, dev mode is not a production installation method,
+but provides a way to get Boundary running with a Postgres database easily. Do
+not use dev mode in a production environment, see the [Production
+Installation](/docs/oss/installing/high-availability) section for
+production-ready deployment methods.
+
+Dev mode brings up a fully-functioning instance of Boundary, including:
+
+- A controller server
+- A worker server
+- A Postgres database
+
+These components should all be considered ephemeral - no data persistence occurs
+across dev mode restarts. Along with these external components, dev mode also
+creates several resources within Boundary to make it easier and faster to
+connect to your first target:
+
+1. A org [scope](/docs/concepts/domain-model/scopes) and a project scope with a
+   default ID of `o_1234567890` and `p_1234567890` respectively.
+1. An [auth method](/docs/concepts/domain-model/auth-methods) with a random
```

Comment:
Three issues here:

1. The login name/password are not random, they are fixed
2. There are actually _two_ users it brings up -- an admin user and an "unprivileged" user
3. Dev mode also brings up an OIDC auth method, with both of these users available through this method as well

Reply:
all great points - updated